### PR TITLE
 Add intro to mashup.jsp #1743 

### LINKED ISSUE #1743: Add intro to mashup.jsp
_From [dam...@gmail.com](https://code.google.com/u/114919422028123491955/) on March 23, 2014 08:55:31_

The page in concern is /dev/mashup.jsp

Add this at the top:

"This page contains a combination of all pages of the TEAMMATES app. It is used for manually inspecting the TEAMMATES UI for visual defects. To load this page, you need to run the test MashupPageUiTest.java."

_Original issue: http://code.google.com/p/teammatespes/issues/detail?id=1746_

### PR DESCRIPTION
**Pull request by zulrich91**
Fixes #1743

### CODE REVIEW

File: .settings/com.google.appengine.eclipse.core.prefs [hunk 1]
```diff
@@ -1,8 +1,9 @@
 eclipse.preferences.version=1
-filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.4.jar|appengine-api-labs.jar|appengine-endpoints-deps.jar|appengine-endpoints.jar|appengine-jsr107cache-1.9.4.jar|datanucleus-appengine-1.0.10.final.jar|datanucleus-core-1.1.5.jar|datanucleus-jpa-1.1.5.jar|geronimo-jpa_3.0_spec-1.1.1.jar|geronimo-jta_1.1_spec-1.1.1.jar|jdo2-api-2.3-eb.jar|jsr107cache-1.1.jar
+filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.18.jar|appengine-api-labs.jar|appengine-endpoints-deps.jar|appengine-endpoints.jar|appengine-jsr107cache-1.9.18.jar|datanucleus-appengine-1.0.10.final.jar|datanucleus-core-1.1.5.jar|datanucleus-jpa-1.1.5.jar|geronimo-jpa_3.0_spec-1.1.1.jar|geronimo-jta_1.1_spec-1.1.1.jar|jdo2-api-2.3-eb.jar|jsr107cache-1.1.jar
```
Comment (kanghj) [inline .settings/com.google.appengine.eclipse.core.prefs]: You need to remove the changes unrelated to this issue from the pull request.

File: src/main/webapp/stylesheets/teammates.css [hunk 2]
```diff
@@ -96,7 +97,10 @@ h2 {
 /*-----------------------------------------------------------------------------------------*/
 
 #header {
-    background-color: black;
+   /* background-color: black; */
+ /*   background-color: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#1e5799), color-stop(50%,#2989d8), color-stop(51%,#207cca), color-stop(100%,#7db9e8)); */
+  /*  background-color: -moz-linear-gradient(top, #1e5799 0%, #2989d8 50%, #207cca 51%, #7db9e8 100%); */
+    background-color: #408080;
```
Comment (kanghj) [inline src/main/webapp/stylesheets/teammates.css]: These changes to the css do not look like they are used for this issue, so you should remove them.

**Comment by kanghj** [comment]
Some changes requested. Can you post a screenshot of the change you made here?

**Comment by zulrich91** [comment]
I have revert the changes to the files that does not concern this Issue !!

**Comment by LowWeiLin** [comment]
@zulrich91 There are still unrelated changes in the pull request you have to remove. You can see them under the files changed tab: https://github.com/TEAMMATES/repo/pull/3157/files

File: src/main/webapp/dev/mashup.jsp [hunk 3]
```diff
@@ -20,7 +20,11 @@
         <script src="/js/common.js" type="text/javascript"></script>
     </head>
     <body>
-        <div class="container theme-showcase">
+    <h1 style="text-align: center">This page contains a combination
+        of all pages of the TEAMMATES app. It is used for manually
+        inspecting the TEAMMATES UI for visual defects. To load this
+        page, you need to run the test MashupPageUiTest.java.</h1>
+    <div class="container theme-showcase">
```
Comment (zulrich91) [inline src/main/webapp/dev/mashup.jsp]: I have removed the files that are associated with this issues , please revise this and let me know if there is any change that i need to revert

File: src/main/webapp/WEB-INF/classes/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/META-INF/
```
Comment (kanghj) [inline src/main/webapp/WEB-INF/classes/.gitignore]: There is still an unrelated change here.

**Comment by kanghj** [comment]
One last unrelated file to remove. Post a screenshot of the changes?

File: src/main/webapp/WEB-INF/classes/.gitignore (hunk 4 above)
Comment (zulrich91) [inline src/main/webapp/WEB-INF/classes/.gitignore]: I can't figure when i modified the file src/main/webapp/WEB-INF/classes/.gitignore, but i found our from the git log that if i have to undo changes to this file , i have to revert the whole commit and refix the error at fresh . Should i go ahead and revert the whole commit ?

Comment (kanghj) [inline src/main/webapp/WEB-INF/classes/.gitignore]: Since there are so many irrelevant commits in this pull request, perhaps make a new pull request? This time, be sure to not include the unrelated commits and files.

Comment (zulrich91) [inline src/main/webapp/WEB-INF/classes/.gitignore]: Ok Jin ,
Will just do as you said .
 thanks.

On Wed, Apr 8, 2015 at 3:04 AM, Kang Hong Jin <notifications@github.com>
wrote:

> In src/main/webapp/WEB-INF/classes/.gitignore
> <https://github.com/TEAMMATES/repo/pull/3157#discussion_r27937773>:
>
> > @@ -0,0 +1 @@
> > +/META-INF/
>
> Since there are so many irrelevant commits in this pull request, perhaps
> make a new pull request? This time, be sure to not include the unrelated
> commits and files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TEAMMATES/repo/pull/3157/files#r27937773>.
>



-- 
ulrich tchuenkam

Comment (zulrich91) [inline src/main/webapp/WEB-INF/classes/.gitignore]: Hi Jin ,

There is this file src/main/webapp/dev/WEB-INF/classes/.gitignore that gets
deleted automatically when ever i edit the file ./mashup.jsp.
This is the same problem i had the last time. I have not deleted this file.
I dont know why this file gets deleted automatically like this .
Just hope you will answer me very soon and let me know what to do. I have
over stayed on this issue .
Thanks

On Wed, Apr 8, 2015 at 12:03 PM, tchuenkam ulrich <ulrichmobi2@gmail.com>
wrote:

> Ok Jin ,
> Will just do as you said .
>  thanks.
>
> On Wed, Apr 8, 2015 at 3:04 AM, Kang Hong Jin <notifications@github.com>
> wrote:
>
>> In src/main/webapp/WEB-INF/classes/.gitignore
>> <https://github.com/TEAMMATES/repo/pull/3157#discussion_r27937773>:
>>
>> > @@ -0,0 +1 @@
>> > +/META-INF/
>>
>> Since there are so many irrelevant commits in this pull request, perhaps
>> make a new pull request? This time, be sure to not include the unrelated
>> commits and files.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/TEAMMATES/repo/pull/3157/files#r27937773>.
>>
>
>
>
> --
> ulrich tchuenkam
>



-- 
ulrich tchuenkam
